build/common.project.props

```diff
@@ -86,6 +86,11 @@
     <LangVersion Condition="'$(DotNetBuildFromSource)' == 'true'">latest</LangVersion>
   </PropertyGroup>
 
+  <!-- NuGet's dev branch always targets the next upcoming version of .NET, so we always use the preview SDK -->
+  <PropertyGroup Condition=" '$(IsEscrow)' != 'true' ">
+    <SuppressNETCoreSdkPreviewMessage>true</SuppressNETCoreSdkPreviewMessage>
```

Comment:
After creating this PR, I had a thought that I could just set this as an environment variable on my machine. If we want to suppress it on CI, we can also use an environment variable there.  Is that better/worse than setting the MSBuild property?

Reply:
I do the env var locally trick, I don't mind suppressing it here.